electron/src/preload.ts

```diff
@@ -0,0 +1,7 @@
+// See the Electron documentation for details on how to use preload scripts:
+// https://www.electronjs.org/docs/latest/tutorial/process-model#preload-scripts
+import { contextBridge, ipcRenderer } from 'electron'
+
+contextBridge.exposeInMainWorld('electronAPI', {
```

Comment:
It took me a bit of time to realize this string, "electronAPI" is the name of the object that is exposed on `window` in the renderer.